### PR TITLE
Integrate linting tools:

### PR DESCRIPTION
**Pull request by domzic**
- Setup Prettier and eslint;
 - Format src code;
 - Add linting job to github actions

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
         id: cache-build
         with:
           path: ./*
-          key: ${{ github.sha }}
+          key: ${{ github.sha }}z
```
Comment (domzic) [inline .github/workflows/github-pages.yml:37]: Remove

File: .editorconfig [hunk 2]
```diff
@@ -7,10 +7,13 @@ trim_trailing_whitespace = true
 
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
 
 [Dockerfile]
-indent_size = 4
+indent_size = 2
```
Comment (rikius) [inline .editorconfig:13]: let's keep it 4 spaces.

File: .editorconfig [hunk 3]
```diff
@@ -7,10 +7,13 @@ trim_trailing_whitespace = true
...
+
+[*.yml]
+indent_size = 2
```
Comment (rikius) [inline .editorconfig:16]: let's keep it 4 spaces.

File: src/App.tsx [hunk 4]
```diff
@@ -1,72 +1,67 @@
...
-import Home from "./pages/Home";
-import About from "./pages/About";
-import PageNotFound from "./pages/PageNotFound";
-import UserProfile from "./pages/UserProfile";
-import AnimalList from "./pages/AnimalList";
-import Animal from "./pages/Animal";
-import PageNotImplemented from "./pages/PageNotImplemented";
+import Container from '@material-ui/core/Container';
 
+import CssBaseline from '@material-ui/core/CssBaseline';
```
Comment (rikius) [inline src/App.tsx:3]: Can we sort all those imports?

**Review by rikius — CHANGES_REQUESTED** [review]
need to fix those small issues, but also it might be merged without them, just remove `z` letter in one of changes.

File: src/App.tsx (hunk 4 above)
Comment (domzic) [reply src/App.tsx:3]: It's sorted

**Comment by saulyz** [comment]
Some files are still with 2 space format.
tsconfig.json, .prettierrc

**Review by saulyz — APPROVED** [review]

**Review by rikius — APPROVED** [review]

**Pull request merged**